src/__tests__/is-safari.test.ts

```diff
@@ -0,0 +1,65 @@
+import isSafari from "../is-safari";
+
+const AGENTS: {
+  [key: string]: string;
+} = require("./helpers/user-agents.json");
+
+describe("isSafari", () => {
+  it("returns true when safari", () => {
```

Comment:
Piggybacking off Holly's comment, we could name this `return true when safari desktop` just so that our tests make it super clear what this method does

Reply:
Renamed the tests to make it more clear